packages/react/src/auto/hooks/useEnumInputController.tsx

```diff
@@ -21,7 +21,7 @@ export const useEnumInputController = (props: {
     control,
     name: path,
   });
-  const [searchValue, setSearchValue] = useState(typeof fieldProps.value === "string" ? fieldProps.value : "");
```

Comment:
Unless you are using the `defaultValues` prop in AutoForm, this would always be initially null, which made this empty anyways in most circumstances